src/scopes/grants/recipientsWithoutTTA.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable import/prefer-default-export */
+import { Op } from 'sequelize';
+import { sequelize } from '../../models';
+
+/*
+  Determine what grants have activity reports within the given date range.
+  From that list of grants, determine which grants are not in that list (no activity).
+  OVERLAPS excludes matches on the start and end dates, so we add and subtract a day.
+*/
+const grantsMissingActivitySql = (beginActivityDate, finishActivityDate) => sequelize.literal(
+  `
+    (WITH activity AS (
+      SELECT
+        g."id" AS used_grant_id
+      FROM "Grants" g
+      JOIN "Goals" gl
+        ON g.id = gl."grantId"
+      JOIN "ActivityReportGoals" arg
```

Comment:
shouldn't this join with ActivityRecipients, why are we using goals?

Reply:
I guess maybe it doesn't matter

Reply:
Good call